requirements.txt

```diff
@@ -6,3 +6,4 @@ pytest-cov==2.6.1
 capturer==2.4
 attrs==19.1.0
 requests==2.24.0
+pymongo==3.11.0
```

Comment:
This is not bundled into the python tracer I guess, right?

Reply:
right :)
This is just dev dependencies